src/index.html

```diff
@@ -1,13 +1,50 @@
+
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>APPLE A1419</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina"
+      >
+
+      <h1 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <h2 class="card__code">Product code: 195434</h2>
+
+      <div class="reting">
+        <div class="reting__stars">
+          <div class="reting__star"></div>
+          <div class="reting__star"></div>
+          <div class="reting__star"></div>
+          <div class="reting__star"></div>
+          <div class="reting__star"></div>
+        </div>
+        <div class="reting__review">Reviews 5</div>
+      </div>
+
+      <div class="card-price">
+        <div class="card-price__price">Price:</div>
+        <div class="card-price__cost">&#36;2,199</div>
+      </div>
```

Comment:
```suggestion
      <div class="card__price">
        <div class="card__price-text">Price:</div>
        <div class="card__price-value">&#36;2,199</div>
      </div>
```

Reply:
elements in BEM should be write in such style `{blockname}__{element}--{modificator}`
so block name is `card`, element - `card__price`